build.sbt

```diff
@@ -523,8 +527,11 @@ lazy val storagePlugin = project
     name                       := "delta-storage-plugin",
     moduleName                 := "delta-storage-plugin",
     libraryDependencies       ++= Seq(
-      akkaHttpXml,
-      alpakkaS3,
+      akkaHttpXml exclude ("com.typesafe.akka", "akka-http_2.13"),
+      alpakkaS3 excludeAll (
+        ExclusionRule(organization = "com.typesafe.akka", name = "akka-stream_2.13"),
+        ExclusionRule(organization = "com.typesafe.akka", name = "akka-http_2.13")
+      ),
       "io.kamon"       %% "kamon-akka-http" % kamonVersion % Provided,
```

Comment:
I've tried that but it does not work, since it removes totally the dependency even on the compile scope and then obviously I get errors loke:
```
[error] /Users/dmontero/github/nexus/delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StoragePlugin.scala:4:34: Unused import
[error] import akka.http.scaladsl.server.Route
```